dozer-ingestion/src/connectors/kafka/debezium/mapper.rs

```diff
@@ -114,9 +116,9 @@ fn convert_value(
                     })
                 }
                 "io.debezium.time.MicroTime" => Ok(Field::Null),
-                "io.debezium.data.Json" => value
-                    .as_str()
-                    .map_or(Ok(Field::Null), |s| Ok(Field::Bson(s.as_bytes().to_vec()))),
+                "io.debezium.data.Json" => value.as_str().map_or(Ok(Field::Null), |s| {
```

Comment:
I'm not sure about this convesion. @karolisg can you have a look?

Reply:
Looks correct. Maybe we can have proper error `JsonValue::from_str(s).unwrap()` instead of unwrap?

Reply:
ok! thanks @karolisg @chubei